makefile

```diff
@@ -1,4 +1,4 @@
-CLANG_FORMAT=node_modules/clang-format/bin/linux_x64/clang-format --style=Google
+CLANG_FORMAT=node_modules/clang-format/bin/darwin_x64/clang-format --style=Google
```

Comment:
Can you please pull from master to get the makefile changes here instead?

Reply:
Done